easybuild/easyblocks/w/wrf.py

```diff
@@ -251,7 +268,7 @@ def test_step(self):
 
             # exclude 2d testcases in non-parallel WRF builds
             if self.cfg['buildtype'] in self.parallel_build_types:
-                self.testcases = [test for test in self.testcases if not "2d_" in test]
+                self.testcases = [test for test in self.testcases if '2d_' not in test]
```

Comment:
It seems to me that the code is right, but the comment is wrong.  When looking at the 2-D testcases, those are really 2-D (x times z or y times z).  Since WRF paralellizes over x/y, it does kind of make sense that parallelization doesn't work in those cases. At least on my machine, I cannot build the 2-D testcases with parallelization.

So my way forward would be to fix the comment to read "exclude 2d testcases in parallel WRF builds".

Reply:
Ok, as long as it's consistent :-)

Reply:
Comment still wrong.